src/components/screens/PointOfInterest.js

```diff
@@ -148,7 +150,7 @@ export const PointOfInterest = ({ data, hideMap, navigation }) => {
           </View>
         )}
 
-        {!hideMap && !!latitude && !!longitude && (
+        {!hideMap && !!latitude && !!longitude && isConnected && isMainserverUp && (
```

Comment:
i think `isConnected` is enough?
the map does not rely on our mainserver, or am i wrong?
if there are `latitude` and `longitude`, we can get a map if online, right?

Reply:
added a comment to this in the code, that explains the value of including the check for `isMainserverUp`

Reply:
we keep both. but both checks are not enough 100%, because the openstreetmap server can be down as well...